hs_core/views/__init__.py

```diff
@@ -1768,6 +1769,11 @@ def create_resource(request, *args, **kwargs):
             full_paths=full_paths,
             auto_aggregate=auto_aggregate,
         )
+    except PartyValidationError as ex:
+        party_message = "Validation issue in Party metadata. " + \
```

Comment:
`Party `is an internal implementation thing. I don't think users would have any idea what it means when it appears as part of the error message.

Reply:
thanks @pkdash, I updated to "Validation issue in Creator or Contributor metadata. One of the profiles contains invalid identifiers: ..."